README.md

```diff
@@ -22,17 +22,17 @@ conda develop /path/to/torchqdynamics
 ## Contribute
 Install developer dependencies:
 ```shell
-pip install isort black
+pip install isort yapf
 ```
 
 Run the following before each commit:
 ```shell
 isort torchqdynamics
-black torchqdynamics
+yapf --style={based_on_style: facebook, column_limit: 88, split_before_named_assigns: False} torchqdynamics
```

Comment:
Weird, yapf should pick the `pyproject.toml` settings without needing to specify all of them. Maybe it's due to

> Should be good now. I had to add a setup.cfg file to work around https://github.com/google/yapf/issues/936.

I think a better fix is just to add `toml` as a dependency.

Reply:
Maybe passing the style here is useless indeed. I was not sure, so I did it anyway :D

Reply:
I think we need `yapf -i -r torchqdynamics`.

Reply:
Ok I'm not quite sure about all this tbh. Can I let you make the proper changes on the pre-commit stuff?